web-common/src/features/entity-management/AddAssetButton.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import Add from "../../components/icons/Add.svelte";
+
+  export let id: string;
+  export let label: string;
+</script>
+
+<button
+  on:click
+  {id}
+  class="pl-2.5 pr-2 py-1 hover:bg-gray-100 w-full text-left flex gap-x-1.5 items-center text-xs"
+>
```

Comment:
Can we use our web-common Button for this or nah?

Reply:
In theory we could have a new Button `variant` that implements the design. Though I figured it was a unique enough button that adding it as a `variant` would be over-engineering. But if we ever re-use this design in other places, then it could make sense to incorporate into a more canonical component.